src/fft-api/facility/fftFacilityService.ts

```diff
@@ -29,26 +29,34 @@ export class FftFacilityService {
 
   constructor(private readonly apiClient: FftApiClient) {}
 
-  public async getFacilityId(tenantFacilityId: string): Promise<string> {
+  public async getFacilityId(tenantFacilityId: string, lenient = false): Promise<string | undefined> {
```

Comment:
maybe it is because i am uneducated, but i needed to google the word "lenient", maybe `relaxed` is a bit better to understand?

Reply:
Didn't know that word either, so a renaming would be great.